addons/kommander/1.3/kommander.yaml

```diff
@@ -49,55 +49,8 @@ spec:
     version: 0.13.10
     values: |
       ---
-      ingress:
-        extraAnnotations:
-          traefik.ingress.kubernetes.io/priority: "2"
-      kommander-licensing:
-        certificates:
-          issuer:
-            name: kubernetes-ca
-            kind: ClusterIssuer
-      kommander-federation:
-        utilityApiserver:
-          minimumKubernetesVersion: 1.16.0
-        certificates:
-          issuer:
-            name: kubernetes-ca
-            kind: ClusterIssuer
-        konvoy:
-          allowUnofficialReleases: false
-        kubefed:
-          controllermanager:
-            annotations:
-              secret.reloader.stakater.com/reload: "kubefed-admission-webhook-serving-cert"
-            webhook:
-              annotations:
-                secret.reloader.stakater.com/reload: "kubefed-admission-webhook-serving-cert"
-      kommander-karma:
-        karma:
-          deployment:
-            annotations:
-              configmap.reloader.stakater.com/reload: kommander-kubeaddons-config
-              secret.reloader.stakater.com/reload: kommander-karma-client-tls
-      kommander-thanos:
-        thanos:
-          query:
-            deploymentAnnotations:
-              secret.reloader.stakater.com/reload: kommander-thanos-client-tls
-      kubeaddons-catalog:
-        ingress:
-          enable: false
-          hostName: catalog.kubeaddons.localhost
-          annotations: {}
-      kommander-ui:
-        podAnnotations: {}
-        #  iam.amazonaws.com/role: xyz
       kubecost:
```

Comment:
can we put this in the chart too? cc @gracedo

Reply:
We need to keep this bit separate (ping me for more info or @DanielMSchmidt, we discussed this offline in the past)

Reply:
I think there was a discussion that both this and the charts repo are open source, so theoretically this can be moved as well 🤔